torchvision/prototype/transforms/_misc.py

```diff
@@ -79,3 +81,28 @@ def __init__(self, dtype: torch.dtype, *types: Type) -> None:
 
     def extra_repr(self) -> str:
         return ", ".join([f"dtype={self.dtype}", f"types={[type.__name__ for type in self.types]}"])
+
+
+class CleanupBoxes(Transform):
+    def _get_params(self, sample: Any) -> Dict[str, Any]:
+        bounding_boxes = query_bounding_box(sample)
+        bounding_boxes_clamped = F.clamp_bounding_box(
+            bounding_boxes, format=bounding_boxes.format, image_size=bounding_boxes.image_size
+        )
+        bounding_boxes_xywh = F.convert_bounding_box_format(
+            bounding_boxes_clamped, old_format=bounding_boxes.format, new_format=features.BoundingBoxFormat.XYWH
+        )
```

Comment:
This might be slow. The clamp already changes format twice internally, so that's the 3rd format change in a row.